### PR TITLE
Added FIELDNAMs, LABLAXISs, and CATDESCs for housekeeping attributes

### LINKED ISSUE #776: CoDICE: Add catdescs and fieldnames to housekeeping CDF attributes
Currently the housekeeping data uses "TBD" for many of the `catdesc` and `fieldname` fields in the CDF attributes. These should be updated to actually store useful information. This could possibly be done programmatically by grabbing the `short_description` and `long_description` fields in the XTCE definition files. Perhaps something like:

```python
for key, value in (packet.header | packet.data).items():
    fieldname = value.short_description
    catdesc = value.long_description
```

### PR DESCRIPTION
**Pull request by bourque**
This PR adds `FIELDNAM`s, `CATDESC`s, and `LABLAXIS`s to the CoDICE L1 housekeeping CDFs.

Closes #776

### CODE REVIEW

File: imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml
Comment (tech3371) [inline imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml]: You will need to move ones with long description to VAR_NOTES. Andriy pointed that out.

File: imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml
Comment (tech3371) [inline imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml:24]: You can add `DEPEND_0` here and then you don't need to add it in `codice_l1a.py`.

File: imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml
Comment (bourque) [reply imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml]: Good catch! It looks like the character limit for `CATDESC` is 80 characters according to the ISTP website, so I will go by that.

**Comment by bourque** [comment]
I think this is good to go once I can get an approved review 😉

File: imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml
Comment (tech3371) [inline imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml]: I am pretty sure `CATDESC` is a required field but you may have set it to `' '` in your `default`. Later you might want to have short description in `CATDESC`

**Review by tech3371 — APPROVED** [review]
One comment. Besides, it looks good

File: imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml
Comment (bourque) [reply imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml]: Good to know, thanks! If I want to add `CATDESC`s for the ones that don't have them, I will have to come up with that myself most likely. But I will add a TODO for those and punt on that for a future PR.

**Pull request merged**
